src/components/swipeActionButton/SwipeButtonText.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { Animated, StyleSheet, TextStyle } from 'react-native';
+
+interface SwipeButtonText {
+  translateX: Animated.Value & { _value?: number };
+  title: string;
+  textStyle?: TextStyle;
+  customText?: JSX.Element;
+  width: number;
+}
+export const SwipeButtonText: FC<SwipeButtonText> = ({
+  translateX,
```

Comment:
- Add custom hook and retrieve values from it, arrange props name in order.
- Move hardcoded values in constant file.